### DISCUSSION TITLE
There's a troubling problem

### DISCUSSION THREAD

**Discussion by xiaoliang6669** [godotengine/godot-proposals#5320] [Scripting]
Why don't you support Chinese variables? Are the managers anti Chinese?

**Comment by Zireael07** [comment]
It's a case of a more general issue (no non-Latin variables are supported) which is already being tracked

**Comment by timothyqiu** [comment]
See https://github.com/godotengine/godot-proposals/issues/916

**Reply by Zireael07** [reply]
Thanks, I was on mobile when I wrote the other answer and I couldn't find the exact #

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/916.
